File: C/URI_judge_1015.c
#include <stdio.h>
#include <math.h>

int main(){
    double x1,y1,x2,y2,r;
    scanf("%lf""%lf",&x1,&y1);
    scanf("%lf""%lf",&x2,&y2);

    r = sqrt(pow(x1 - x2, 2) + pow(y1 - y2, 2));

    printf("%.4lf",r);
}

File: Python/URI_judge_1015.py
import math

x = input().split()
y = input().split()

x1,y1 = map(float,x)
x2,y2 = map(float,y)

r = math.sqrt(pow(x1 - x2,2) + pow(y1 - y2, 2))

print(f"{r:.4f}")